README.md

```diff
@@ -1 +1,37 @@
-# ctd-omega-editorial-frontend
\ No newline at end of file
+# ctd-omega-editorial-frontend
```

Comment:
Can we call this Pan-Archival Catalogue Editorial Frontend instead of just using the repository name please. This is in line with what we have done in other repositories.

Could we also please add the following badges just below here:-
```
[![Build Status](https://github.com/nationalarchives/kettle-test-framework/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/nationalarchives/kettle-test-framework/actions/workflows/ci.yml)
[![Scala 2.13](https://img.shields.io/badge/scala-2.13-red.svg)](http://scala-lang.org)
[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://opensource.org/licenses/MIT)
```